spec/type.dd

```diff
@@ -476,36 +479,42 @@ $(H3 $(LNAME2 vrp, Value Range Propagation))
 
     $(SPEC_RUNNABLE_EXAMPLE_COMPILE
     ---
-    extern char c;
-    // min is c.min + 100 > short.min
-    // max is c.max + 100 < short.max
-    short s = c + 100; // OK
-
-    extern int i;
-    ubyte j = i & 0x3F; // OK, 0 ... 0x3F
-    //ubyte k = i & 0x14A; // error, 0x14A > ubyte.max
-    ushort k = i & 0x14A; // OK
-
-    extern ubyte b;
-    k = i & b; // OK, 0 ... b.max
-    //b = b + b; // error, b.max + b.max > b.max
-    s = b + b; // OK, 0 ... b.max + b.max
+    void fun()
+    {
+        extern char c;
```

Comment:
extern doesn't make sense on local variables

Reply:
Now changed to use parameters instead.